otter/test/convergence/test_service.py

```diff
@@ -1366,6 +1374,41 @@ def plan(*args, **kwargs):
             perform_sequence(self.get_seq() + sequence, self._invoke(plan)),
             ConvergenceIterationStatus.Stop())
 
+    def test_launch_stack_config(self):
+        """Using a launch_stack launch config works."""
+        lc = {'args': {'stack': {'stack_name': 'foo'}},
+              'type': 'launch_stack'}
+
+        self.manifest = {
+            'state': self.state,
+            'launchConfiguration': lc,
+        }
+
+        self.gsgi_result = (self.group, self.manifest)
+        self.gacd_runner = lambda i: {'stacks': []}
+
+        def plan(*args, **kwargs):
+            return []
+
+        executor = attr.assoc(launch_stack_executor,
+                              gather=intent_func('gacd'),
+                              plan=plan)
+
+        seq = [
+            parallel_sequence([]),  # No steps to log (log_steps)
+            (Log(msg='execute-convergence', fields=mock.ANY), noop),
+            (Log(msg='execute-convergence-results', fields=mock.ANY), noop),
+            (ModifyReference(self.waiting,
+                             match_func(pmap({self.group_id: 43}),
+                                        pmap())),
+             dispatch(reference_dispatcher)),
+        ]
+        eff = execute_convergence(
```

Comment:
Non-blocking: There is `_invoke` function in the class to call this. Would it be possible to modify that to work with this test? One thing I can think of is `_invoke` taking `executor` as an argument that if not provided defaults to launch_server executor.